test/utils.cpp

```diff
@@ -671,4 +671,40 @@ at::Tensor atBiasEpilogue(const at::Tensor& tensor, const at::Tensor& bias) {
   return tensor.add(bias.unsqueeze(-1));
 }
 
+size_t getRandomSeed(bool aten) {
```

Comment:
Should we have a `getRandomSeed` and a `getATenRandomSeed`? Or at least make the `bool aten` a template parameter, instead of a function parameter. Currently, if I set `TEST_ATEN_RANDOM_SEED`, the other tests will use the same seed for `srand`.

Reply:
That would be better definitely. The parameter is confusing. Will change it.

Reply:
I split it into `getCRandomSeed` and `getATenRandomSeed`